content/en/serverless/step_functions/installation.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Install Serverless Monitoring for AWS Step Functions
+kind: documentation
+further_reading:
+    - link: '/serverless/configuration/'
+      tag: 'Documentation'
+      text: 'Configure Serverless Monitoring'
+    - link: "/integrations/amazon_lambda/"
+      tag: "Documentation"
+      text: "AWS Lambda Integration"
+---
+
+### Requirements
+* States must last less than 15 minutes for full traces.
+* [Associated Lambda tracing][1] is supported for Node.JS and Python runtimes.
+
+### Setup
+
+{{< tabs >}}
+{{% tab "Serverless Framework" %}}
```

Comment:
@sumedham maybe people use both lambda and SF (quite common) would now ask should I use the forwarder for both lambda and SF? perhaps we should give them a note that we do recommend using extension for lambda while forwarder for SF?